_sub/security/aws-backup-for-rds/main.tf

```diff
@@ -0,0 +1,84 @@
+# AWS Vault Creation
+resource "aws_backup_vault" "vault" {
+  count                     = var.deploy && var.vault_name != "" ? 1 : 0
```

Comment:
Please remove count on the resource, I think we can create it when if we deploy the module.

Reply:
Done.